### PR TITLE
Bike drift and Turret Implementation 

### PR DESCRIPTION
**Pull request by Tomerd-afk**
This is the branch that has my bike drift and turret implementation.

### CODE REVIEW

**Comment by Tomerd-afk** [comment]
Bug: turret runs out of bullets if fire rate too high.

File: Assets/Scripts/Bullets/BasicRifleBullet.cs [hunk 1]
```diff
@@ -21,7 +21,14 @@ private void OnTriggerEnter(Collider other)
             tracerHealth.TakeDamage(damageDealt);
             //Debug.Log("Hit Player!");
         }
+        if (other.gameObject.tag == "Enemy")
```
Comment (toryjleo) [inline Assets/Scripts/Bullets/BasicRifleBullet.cs:24]: Remove this section

Comment (toryjleo) [reply Assets/Scripts/Bullets/BasicRifleBullet.cs:24]: You can add another version for the rifle if that is what you are doing here

Comment (Tomerd-afk) [reply Assets/Scripts/Bullets/BasicRifleBullet.cs:21]: I have now added a new Turret Bullet

**Comment by toryjleo** [comment]
Please make enabling/disabling the turret an easy thing

**Pull request closed without merging**